backend/server.js

```diff
@@ -0,0 +1,226 @@
+require('dotenv').config({ path: './backend/.env' })
+const express = require('express')
+const app = express()
+const port = 3001
+
+
+const mongoose = require('mongoose')
+const nacl = require('tweetnacl')
+nacl.util = require('tweetnacl-util')
+const axios = require('axios')
+const Web3 = require('web3')
+const stringify = require('fast-json-stable-stringify');
+
+const { v4: uuidv4 } = require('uuid');
+const CryptoJS = require('crypto-js')
+
+const EthereumTx = require('ethereumjs-tx');
+
+// TODO: Change provider to main net.
+const web3 = new Web3(process.env.WEB3_PROVIDER_URI);
+
+web3.eth.defaultAccount = process.env.ETH_ADDRESS
+
+/**
+ * On startup, print balance on account as a sanity check for Chain config.
+ */
+const printBalance = async () => {
+  const myBalanceWei = await web3.eth.getBalance(web3.eth.defaultAccount)
+  const myBalance = web3.utils.fromWei(myBalanceWei, 'ether') // 'ether' is just the chain token.
+  console.log(`Your wallet balance is currently ${myBalance} XDai`)
+}
+
+// TODO: Setup password for database
+mongoose.connect('mongodb://localhost/d4e', {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+})
+
+const CONTEXT = 'DollarForEveryone'
+const BRIGHTID_NODE_DOMAIN = process.env.BRIGHTID_NODE_DOMAIN
+const BRIGHTID_NODE_URL = BRIGHTID_NODE_DOMAIN + '/brightid/v5'
+
+const userSchema = new mongoose.Schema({
+  contextId: String,
+  verified: Boolean,
+  creationDate: Date,
+  receivedDate: Date,
+  address: String
+})
+
+/**
+ * From https://github.com/BrightID/BrightID/blob/9a9570dd0b1a39aafa770e1f3bf745259b023bc5/BrightID/src/utils/encoding.js#L37
+ */
+function b64ToUrlSafeB64 (s) {
+  const alts = {
+    '/': '_',
+    '+': '-',
+    '=': '',
+  };
+  return s.replace(/[/+=]/g, (c) => alts[c]);
+}
+
+function hash (data)  {
+  const h = CryptoJS.SHA256(data);
+  const b = h.toString(CryptoJS.enc.Base64);
+  return b64ToUrlSafeB64(b);
+};
+
```

Comment:
```suggestion
```

It seems this function is no longer used now that we're using stable-json-stringify and nacl.sign to encode operations.